packages/flutter/test/widgets/list_wheel_scroll_view_test.dart

```diff
@@ -1014,6 +1014,7 @@ void main() {
     await tester.pumpAndSettle();
     expect(controller.offset, 500.0);
 
+    // Test failing here.
```

Comment:
This one test is still failing because I have changed the way that children are spawned. Here, the code is looking for a child that is too far in the viewport that I haven't spawned yet => error.
I don't know whether I should change this test, or spawn all children when childCount is valid (so that the test will pass. What do you think @xster ?

Reply:
@goderbauer to confirm.

My intuition reading the API doc is that we should change this test to only try to look for children that are in range of the view port.

Reply:
What @xster says sounds reasonable. You can also try setting skipOffstage to false on the finder to get to (some of) the invisible children: https://master-docs-flutter-io.firebaseapp.com/flutter/flutter_test/Finder/skipOffstage.html